Makefile

```diff
@@ -4,7 +4,15 @@ all: help
 
 boots: cmd/boots/boots ## Compile boots for host OS and Architecture
 
-bindata: ipxe/bindata.go ## Build and generate embedded iPXE binaries
```

Comment:
Why make the change here rather than in rules.mk?

Reply:
oh, maybe you mean a completely new target in rules.mk?

Reply:
This feels like a hack rather than a proper solution, but maybe that's what we need right now.
I'm not going to block this change, but I also feel uncomfortable explicitly approving it.

Reply:
Would you mind helping me understand why you feel this is a hack?

Reply:
sure I will move this to rules.mk. 

In my opinion, I don't believe having the Makefile only contain simple target definitions reduces or removes any complexity in our overall Make structure. It only moves the complexity and adds more code.